artifact/remote/testdata/test.git/hooks/applypatch-msg.sample

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
```

Comment:
Do we really need all hooks in the `test.git` repo? I was able to run tests just fine without them.

Reply:
Ah, it is unnecessary. I've just run `git init`. I'll remove them.

Reply:
I removed them https://github.com/aquasecurity/fanal/pull/112/commits/1dd8e2e51167244d38e40861f3dbc7fd191c1196, but the test still passes. Thanks!